src/views/settings.html

```diff
@@ -39,8 +39,8 @@ <h1>Daemon Connection:</h1>
                 <div class="form__field">
                   <h1>Daemon Management:</h1>
                   </div>
-                  <input type="radio"><span class="pad-left-10">Turn Daemon On</span><br>
-                  <input type="radio"><span class="pad-left-10">Turn Daemon Off</span><br><br>
+                  <input id="DaemonOn" type="radio"><span class="pad-left-10">Turn Daemon On</span><br>
```

Comment:
These radio buttons do not seem to be functioning correctly. Currently both can be selected at the same time. Expected behavior is either one should be selected or the other